src/app/doubtfire-angularjs.module.ts

```diff
@@ -220,7 +220,7 @@ import {FUnitTaskListComponent} from './units/states/tasks/viewer/directives/f-u
 import {FTaskDetailsViewComponent} from './units/states/tasks/viewer/directives/f-task-details-view/f-task-details-view.component';
 import {FTaskSheetViewComponent} from './units/states/tasks/viewer/directives/f-task-sheet-view/f-task-sheet-view.component';
 import {TasksViewerComponent} from './units/states/tasks/tasks-viewer/tasks-viewer.component';
-
+import {GroupSetSelectorComponent} from '/workspace/doubtfire-web/src/app/groups/group-set-selector/group-set-selector.component';
```

Comment:
you'll need to change this to a relative import by removing the `/workspace/doubtfire-web/src/app/` and import it with `./groups`